packages/devtools_app/lib/src/shared/http/http_request_data.dart

```diff
@@ -55,14 +56,33 @@ class DartIOHttpRequestData extends NetworkRequest {
   HttpProfileRequestRef _request;
 
   final int wrapperId;
+  bool isOutStanding = false;
 
-  Future<void> getFullRequestData() {
-    return serviceManager.service!
-        .getHttpProfileRequest(
-          _request.isolateId,
-          _request.id.toString(),
-        )
-        .then((updated) => _request = updated);
+  /// A notifier that changes when the request data, or it's response body
+  /// changes.
+  ValueNotifier<int> updateCount = ValueNotifier<int>(0);
```

Comment:
why is this notifier necessary? is this just a hack to make sure the response view rebuilds when the response comes available?

Reply:
maybe instead of using a count notifier where we don't actually use the count, we can use a bool value notifier `isFetching` or `isComplete` or something that we can change to notify the request view to rebuild. Unless I'm misunderstanding something and we actually expect this notifier to be notified more than once?

Reply:
Oh I see now it is used from merge too. Maybe rename to `requestUpdatedNotifier` to make it clear that we aren't actually counting anything.